specs/_features/eip7594/das-core.md

```diff
@@ -221,6 +224,40 @@ def get_data_column_sidecars(signed_block: SignedBeaconBlock,
     return sidecars
 ```
 
+#### `get_extended_sample_count`
+
+```python
+def get_extended_sample_count(allowed_failures: uint64) -> uint64:
+    assert 0 <= allowed_failures <= NUMBER_OF_COLUMNS // 2
+
+    def math_comb(n: int, k: int) -> int:
+        if not 0 <= k <= n:
+            return 0
+        r = 1
+        for i in range(min(k, n - k)):
+            r = r * (n - i) // (i + 1)
+        return r
+
+    def hypergeom_cdf(k: uint64, M: uint64, n: uint64, N: uint64) -> float:
```

Comment:
Could you please note what is k, M, n, and N?

https://en.wikipedia.org/wiki/Hypergeometric_distribution

I noticed that M is N, n is K, and N is n in Wikipedia.

How about changing the parameter names consistently with Wikipedia? i.e., k, N, K, and n.

I think the names from Wikipedia quite make sense. k, K mean success objects and n, N mean all objects.

Lower case k and n mean the drawn objects while Upper case K and N mean all objects.